GCRCatalogs/alphaq.py

```diff
@@ -243,6 +268,12 @@ def _collect_native_quantities(name, obj):
         if catalog_version < StrictVersion('3.0'):
             self._quantity_modifiers.update({
                 'host_id': 'hostIndex',
+                'ellipticity_1_true':       'morphology/totalEllipticity1'),
```

Comment:
Invalid syntax here. Remove `)` in this and the following lines.

Additionally, need to add `'position_angle_true':      'morphology/positionAngle',` here too.